io.github.louis77.tuner.yml

```diff
@@ -0,0 +1,145 @@
+  - "--share=network"
+  - "--metadata=X-DConf=migrate-path=/com/github/louis77/tuner/"
+  - "--socket=pulseaudio"
+  - "--talk-name=org.freedesktop.Notifications"
+  - "--own-name=org.mpris.MediaPlayer2.Tuner"
+  - "--env=PATH=/usr/bin:/app/bin:/app/vala/bin"
+  - "--env=LD_LIBRARY_PATH=/app/vala/lib:/app/lib:/lib"
+
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/share/pkgconfig"
+  - "/share/aclocal"
+  - "/man"
+  - "/share/man"
+  - "/share/gtk-doc"
+  - "/share/vala"
+  - "*.la"
+  - "*.a"
+
+modules:
+- name: vala
+  buildsystem: simple
+  build-commands:
+    - install -d /app/vala
+    
+- name: vala-mod
+  buildsystem: meson
+  builddir: _build
+  modules:
+    - name: graphviz
+      buildsystem: autotools
+      config-opts:
+        - --prefix=/app
+      sources:
+        - type: archive
+          url: "https://gitlab.com/graphviz/graphviz/-/archive/10.0.1/graphviz-10.0.1.tar.gz"
+          sha256: "28f452ef1cb12288c8758a62f8c3fcfefdb91b251f7aae61d0d703f851bde931"
+    - name: valac
+      buildsystem: autotools
+      config-opts:
+        - --prefix=/app
+      cleanup:
+        - /share/info
+        - /share/man
+        - /share/doc
+        - /include
+      sources:
+        - type: archive
+          url: "https://download.gnome.org/sources/vala/0.56/vala-0.56.17.tar.xz"
+          sha256: "26100c4e4ef0049c619275f140d97cf565883d00c7543c82bcce5a426934ed6a"
+
+- name: libsoup
+  buildsystem: meson
+  builddir: true
+  config-opts:
+    - "--buildtype=release"
+    - "-Dvapi=enabled"
+    - "-Dtests=false"
+    - "-Dgssapi=disabled"
+    - "-Dsysprof=disabled"
+  sources:
+    - type: git
+      url: https://gitlab.gnome.org/GNOME/libsoup.git
+      tag: 3.6.1
+      commit: 8b46a93bc1cbadb22dcdbb6844d9616723376535
+
+- name: taglib
+  buildsystem: cmake-ninja
+  config-opts:
+    - "-DBUILD_SHARED_LIBS=ON"
+    - "-DCMAKE_BUILD_TYPE=Release"
+  sources:
+    - type: git
+      url: https://github.com/taglib/taglib.git
+      tag: v2.0.2
+      commit: e3de03501ff66221d1f1f971022b248d5b38ba06
+
+- name: gee
+  buildsystem: autotools
+  build-options:
+    cflags: -Wno-error=incompatible-pointer-types
+  cleanup: ["*"]
+  sources:
+    - type: archive
+      url: https://download.gnome.org/sources/libgee/0.20/libgee-0.20.6.tar.xz
+      sha256: 1bf834f5e10d60cc6124d74ed3c1dd38da646787fbf7872220b8b4068e476d4d
+
+- name: granite
+  buildsystem: meson
+  config-opts:
+    - "--buildtype=release"
+  sources:
+    - type: archive
+      url: https://github.com/elementary/granite/archive/refs/tags/6.2.0.tar.gz
+      sha256: 067d31445da9808a802fca523630c3e4b84d2d7c78ae547ced017cb7f3b9c6b5
+
+- name: tuner
+  buildsystem: meson
+  build-options:
+    cflags: "-g"
+    cxxflags: "-g"
+    buildtype: "debug"
+  config-opts:
+  - "--buildtype=debug"
+  
+  post-install:
+  - install -Dm644 /app/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.svg  -t /app/share/icons/hicolor/128x128/apps/
+  sources:
+  - type: dir
```

Comment:
never use source `type: dir`

Reply:
Had the local build and flathub manifest both uploaded. Now corrected
Original app is **com.github.louis77.tuner**
Renaming as trying to address flathub quality issues and was told renaming is not a big deal.
Apologies for errors - it was a long day of churning through build config and I was more tired than I was aware.

Reply:
renaming IS a big deal. Always.

Reply:
We can abort if you would like?

Reply:
> Renaming as trying to address flathub quality issues 

which quality issue is asking you to rename the ID? Then it should be fixed.

Reply:
That page is specifically for "new" submissions, existing apps are grandfathered into the requirements.

Reply:
OK.
So what should I do? What do you want?